cql3/operation.cc

```diff
@@ -357,4 +358,9 @@ operation::element_deletion::prepare(data_dictionary::database db, const sstring
     abort();
 }
 
+void
+operation::prepare_for_raft(raft::broadcast_tables::update_query&) const {
+    throw raft::broadcast_tables::unsupported_operation_error{};
```

Comment:
Is there no easy way to provide some details at this place (e.g. maybe there's a way to pretty-print (or ugly-print) the operation that we attempted to execute)?

Reply:
Also `prepare_for_raft` should eventually become a pure virtual function in `operation` and it should be implemented in every type of `operation`.

Please live a FIXME which tells us that, so the reader of the code knows this state is temporary (there is no inherent reason that we throw unsupported here other than it's WIP).

Reply:
> Is there no easy way to provide some details at this place (e.g. maybe there's a way to pretty-print (or ugly-print) the operation that we attempted to execute)?

We can use the `_e` attribute ( `std::optional<expr::expression>`), but it will work
> only by accident, because all of our operations happen to have on expression.

or use a column definition, but then we won't get the information about a type of the operation.

Reply:
> Please live a FIXME which tells us that, so the reader of the code knows this state is temporary (there is no inherent reason that we throw unsupported here other than it's WIP).

Done